src/renderer/pages/App.js

```diff
@@ -82,8 +82,9 @@ class App extends React.Component {
     return (
       <div key='errors'
         className={'error-popover ' + (hasErrors ? 'visible' : 'hidden')}>
-        <div key='title' className='title'><FormattedMessage id='error'
-          defaultMessage='Error'/></div>
```

Comment:
Why do we need `defaultMessage` here (and the `||` defaults above)?

IMO English could just be one of the translation languages (and the default, in case we don't have translations for the user's system language yet)